site/docs/pages/is-entrypoint.mdx

```diff
@@ -0,0 +1,34 @@
+# `isValidAAEntrypoint`
+
+The `isValidAAEntrypoint` utility is designed to verify if the entrypoint address is a valid v6 entrypoint.
+
+## Usage
+
+:::code-group
+
+```tsx [code]
+import { isValidAAEntrypoint } from '@coinbase/onchainkit';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+
+const isValid = isValidAAEntrypoint({ entrypoint: ENTRYPOINT_ADDRESS_V06 });
+
+if (isValid) {
+  console.log('The entrypoint is valid.');
+} else {
+  console.log('Invalid entrypoint.');
+}
+```
+
+```ts [return value]
+true;
+```
+
+:::
+
+## Returns
+
+[`isValidAAEntrypointtResponse`](/types#isvalidaaentrypointtresponse)
```

Comment:
As the response is just a boolean we don't need a custom type, just write boolean like we did in 
https://onchainkit.xyz/is-base#returns

Reply:
Updated